ebpf/aya-ebpf-bindings/src/aarch64/bindings.rs

```diff
@@ -1716,6 +1716,13 @@ pub struct bpf_sock_tuple__bindgen_ty_1__bindgen_ty_2 {
     pub sport: __be16,
     pub dport: __be16,
 }
+pub mod tcx_action_base {
```

Comment:
You need to add bindings for powerpc64 and s390x.

Reply:
This should be taken care of automagically by actions